cli/src/scanner.rs

```diff
@@ -73,7 +73,16 @@ impl FileSet {
                 }
             }
         }
-        let codeowners = codeowners_file.map(|path| codeowners::from_path(path.as_path()));
+        let codeowners = codeowners_file.and_then(|path| {
+            if let Ok(owners) = codeowners::from_path(path.as_path()) {
+                return Some(owners);
+            }
+            log::debug!(
+                "Found CODEOWNERS file `{}`, but couldn't parse it.",
```

Comment:
Should be `error!`?

Reply:
Also, let's log the parsing error.

Reply:
I fine with that. Do we want to show this error to the users though? We plan to follow up by uploading the CODEOWNERS file regardless so that we can eventually post-process

Reply:
Also, the parsing error isn't useful—it's just a `PatternError` from the `glob` crate. That won't help anyone understand why their file isn't correct.